overlord/ifacestate/ifacestate_test.go

```diff
@@ -861,6 +886,100 @@ slots:
 	check(change)
 }
 
+func (s *interfaceManagerSuite) TestConnectTaskCheckDeviceScopeNoStore(c *C) {
```

Comment:
You had nice comments for the AutoConnect tests, below. It would be nice to have similar comments here. Not a blocker.

Reply:
again, this test class didn't have comments, I'll see what I can do though